### ISSUE TITLE
Issue with downloading - user and licence key not configurable

### ISSUE THREAD

**Issue by jonathanpardon** [vaersaagod/geomate#62]
The maxmind documentation show that you need to configure a user and license key to be able to download the files.
This is not configurable in Geomate, so the download fails. Maybe the download system was updated recently?

From https://dev.maxmind.com/geoip/updating-databases#directly-downloading-databases

curl -O -J -L -u YOUR_ACCOUNT_ID:YOUR_LICENSE_KEY 'https://download.maxmind.com/geoip/databases/GeoIP2-City-CSV/download?suffix=zip'
OR 
wget --content-disposition --user=YOUR_ACCOUNT_ID --password=YOUR_LICENSE_KEY 'https://download.maxmind.com/geoip/databases/GeoIP2-City-CSV/download?suffix=zip'

**Comment by aelvan** [comment]
Looks like the format described in the docs still works for me:

```
'countryDbDownloadUrl' => 'https://download.maxmind.com/app/geoip_download?edition_id=GeoLite2-Country&license_key=YOUR_LICENSE_KEY&suffix=tar.gz',
'cityDbDownloadUrl' => 'https://download.maxmind.com/app/geoip_download?edition_id=GeoLite2-City&license_key=YOUR_LICENSE_KEY&suffix=tar.gz',
```

**Comment by jonathanpardon** [comment]
Yes ok, these url's work, but are not mentioned anymore in the Maxmind documentation.
Would feel more comfortable to have the new url's with basic auth curl in the future.

Thanks for the support.
